pages/documentation.py

```diff
@@ -0,0 +1,176 @@
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+from dash_html_components import *
+from textwrap import dedent
+
+sub = lambda letter, number: [Em(f"{letter}"), Sub(f"{number}")]
+E1 = sub('E', 1)
+E2 = sub('E', 2)
+E12 = sub('E', "1,2")
+X1 = sub("X", 1)
+X2 = sub("X", 2)
+x1 = sub("x", 1)
+x2 = sub("x", 2)
+
+
+def layout(*args, **kwargs):
+    return dbc.Col(width={'size': 8, 'offset': 2}, className='text-justify documentation', children=[
+        H1("Documentation", className='display-4 text-center my-5'),
+
+        H2("Screen Design"),
+        P(["Information on screen design, cell lines, compounds and quality control can be downloaded here: ", Br(),
+                A("Download Screen Information", href="/downloads/screen_information_20190211.pdf", target="_blank", rel="noopener noreferrer"),
+                " (PDF, 614kB)."]),
+        H2("Single agent & combination response"),
+        P("Single agent responses are normalised to controls and fitted. Single agent maximum effect (MaxE) is derived from the fit of the single agent response at the highest screened concentration (fitted for individual replicates if available)."),
+        P("Measured inhibition of the combination wells is also normalised to controls. The combination maximum effect (combo_MaxE) corresponds to the second highest measured data point. There is currently no combination surface fit available in MatrixExplorer v1."),
+
+        H2("Synergy reference models"),
+        H4("Highest Single Agent (HSA)"),
+        P(["According to the HSA model, a combination of Drug 1 and Drug 2 is classified as synergistic if the effect of the combination (", *E12, ") is larger than the effect of either Drug 1 (", *E1, " or Drug 2 (", *E2, ") alone (whichever is larger)."]),
```

Comment:
is larger than the effect of either Drug 1 (", *E1, " or Drug 2 (", *E2, ") alone (whichever is larger)."])

is larger than the effect of either Drug 1 (", *E1, ") or Drug 2 (", *E2, ") alone (whichever is larger)."])